### DISCUSSION TITLE
FPOS Logo, Ethernet Connection, SSO Internet

### DISCUSSION THREAD

**Discussion by JustinKubicz** [guysoft/FullPageOS#583] [Q&A]
#### What were you doing?
Trying to use FPOS to display a weather radar
[Please be as specific as possible here]

#### What did you expect to happen?
Connect to the Website and display
#### What happened instead?
Showing FPOS logo and cursor, I understand this means it's not connected, however I have it on ethernet. It's a business network that I work for that requires a sign on using a browser the first time. 
#### Was there an error message displayed? What did it say?
No
#### Version of FullPageOS?
Not sure where to find /etc? I just used the Nightly build of RaspberryPi Imager that was released in March 2024, I tried the stable as well with the same issue. 
[Can be found in /etc/fullpageos_version ALWAYS INCLUDE.]

#### Screenshot(s) showing the problem:

[If applicable. Always include if unsure or reporting UI issues.]

**Comment by guysoft** [comment]
You can disable the network check and then it would work. You could then sign in once, and if your network is set to remember devices you should be able to put the network check back in place. Use ctrl +t to open a new tab once the browser loaded.


Put in a file called ``/boot/firmware/check_for_httpd`` the word ``disabled``.
Code that checks that is here:
https://github.com/guysoft/FullPageOS/blob/devel/src/modules/fullpageos/filesystem/opt/custompios/scripts/run_onepageos#L9

Also converting to a discussion sine this is not a bug.

**Reply by JustinKubicz** [reply]
I'll try that tomorrow, thanks for your swift response :)
